routes/auth_routes.js

```diff
@@ -0,0 +1,49 @@
+var express = require('express');
+var https = require('https');
+var clientId = process.env.CLIENT_ID;
+var clientSecret = process.env.CLIENT_SECRET;
+
+var authRouter = module.exports = express.Router();
+
+authRouter.get('/', function(req, res) {
+  res.redirect('https://github.com/login/oauth/authorize/?client_id=' + clientId);
+});
```

Comment:
Should this be middleware instead of a router?  If so, the user could be redirected to whatever page they were trying to access after a successful oauth login.